test/integration/tasks/create-new-environment.ts

```diff
@@ -33,5 +33,26 @@ export default async () => {
 
   console.log(`New "${environmentId}" environment is created`)
 
+  console.log(`Check if test alias "${testAliasId}" is available`)
+
+  try {
+    await space.getEnvironmentAlias(testAliasId)
+  } catch (e) {
+    if (JSON.parse(e.message).status === 404) {
+      console.log(`"${testAliasId}" is not available, lets try to create it`)
+      try {
+        const baseEnvironment = await space.getEnvironment('test-base')
+        await space.createEnvironmentAliasWithId(testAliasId, {
+          environment: baseEnvironment as any,
+        })
+        console.log(`New alias "${testAliasId}" has been created`)
+      } catch (e) {
+        console.log('Could not create the alias', e)
```

Comment:
should we rethrow here? Seems like the test run will fail if the Alis is missing

Reply:
Yes that actually makes a lot of sense 👍